assets/thanos-ruler/service-account.yaml

```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  annotations:
-    serviceaccounts.openshift.io/oauth-redirectreference.thanos-ruler: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"thanos-ruler"}}'
+  annotations: {}
```

Comment:
I'm afraid we'll need to do it in two steps https://github.com/openshift/cluster-monitoring-operator/blob/e3ea72fdc49d14443583418002e25b136f69d692/Documentation/development.md?plain=1#L52
(just realized that section is in the wrong place, I'll move, tell me if you have any preference.)
(I didn't test/check this, ignore if you did)

Reply:
thanks, updated!

Reply:
I think `-` should be a suffix :)
I'll add an example as well. I think`kubectl label` uses the same syntax.